internal/commands/vaultsecrets/secrets/secrets.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package secrets
+
+import (
+	"github.com/hashicorp/hcp/internal/pkg/cmd"
+	"github.com/hashicorp/hcp/internal/pkg/flagvalue"
+	"github.com/hashicorp/hcp/internal/pkg/heredoc"
+	"github.com/hashicorp/hcp/internal/pkg/vaultsecrets"
+)
+
+var (
+	// appName is the name of the Vault Secrets application. If not specified,
+	// then the value from the active profile will be used.
+	appName string
+)
+
+func NewCmdSecrets(ctx *cmd.Context) *cmd.Command {
+	cmd := &cmd.Command{
+		Name:      "secrets",
+		ShortHelp: "Manage Vault Secrets application secrets.",
+		LongHelp: heredoc.New(ctx.IO).Must(`
+		The {{ template "mdCodeOrBold" "hcp vault-secrets secrets" }} command group lets you
+		manage Vault Secrets application secrets.
+		`),
+		Flags: cmd.Flags{
+			Persistent: []*cmd.Flag{
+				{
+					Name:         "app-name",
+					DisplayValue: "NAME",
+					Description:  "The name of the Vault Secrets application. If not specified, then the value from the active profile will be used.",
+					Shorthand:    "a",
```

Comment:
nit: Replace sentence with: `If not specified, the value from the active profile will be used.`